sklearn/mixture/tests/test_gaussian_mixture.py

```diff
@@ -1387,3 +1389,33 @@ def test_gaussian_mixture_single_component_stable():
     X = rng.multivariate_normal(np.zeros(2), np.identity(2), size=3)
     gm = GaussianMixture(n_components=1)
     gm.fit(X).sample()
+
+
+def test_gaussian_mixture_all_init_does_not_estimate_gaussian_parameters(
+    monkeypatch,
+    global_random_seed,
+):
+    """When all init parameters are provided, the Gaussian parameters are not estimated.
```

Comment:
@jiawei-zhang-a Could you kindly check if this line is more than 80 characters long and split it up if it is in order to maintain the standards followed?